config/settings.json

```diff
@@ -52,6 +52,32 @@
       "methods": ["GET", "POST", "PUT", "DELETE"],
       "encode": false
     },
+    "projections": [
+      {
+        "code": "EPSG:3031",
+        "extent": [
+          -12369000,
+          -12369000,
+          12369000,
+          12369000
+        ],
+        "isGlobal": false,
+        "proj4": "+proj=stere +lat_0=-90 +lat_ts=-71 +lon_0=0 +k=1 +x_0=0 +y_0=0 +datum=WGS84 +units=m +no_defs",
+        "title": "Antarctic Polar Stereographic"
+      },
+      {
+        "code": "EPSG:3413",
+        "extent": [
+          -12369000,
+          -12369000,
+          12369000,
+          12369000
+        ],
+        "isGlobal": false,
+        "proj4": "+proj=stere +lat_0=90 +lat_ts=70 +lon_0=-45 +k=1 +x_0=0 +y_0=0 +datum=WGS84 +units=m +no_defs",
+        "title": "NSIDC Sea Ice Polar Stereographic North"
+      }
+    ],
```

Comment:
Not positive we want to include these. I hardcode the `proj4` defs later on so that the polar split function can work even if they are not provided here (and they won't show up in the UI).